specification/2.0/schema/skin.schema.json

```diff
@@ -11,7 +11,8 @@
         },
         "skeleton": {
             "allOf": [ { "$ref": "glTFid.schema.json" } ],
-            "description": "The index of the node used as a skeleton root. When undefined, joints transforms resolve to scene root."
+            "description": "The index of the node used as a skeleton root.",
+            "gltf_detailedDescription": "The index of the node used as a skeleton root. The node must be the closest common root of the joints hierarchy or a direct or indirect parent node of the closest common root."
```

Comment:
This no longer specifies what happens when the `skeleton` property is omitted... should we say that the joints must have a common root in a scene node? That may be a breaking change compared to `resolves to scene root` from before. Or should we say that implementations may try to find a common root, which may not exist, or may use the scene root?

Reply:
The previous description has never defined what "scene root" was, so I don't see this as a breaking change. We've also agreed that `skeleton` doesn't affect node transforms. The updated readme says that the common root (node) must exist.

Reply:
It seems technically possible that assets may exist that do not have a common root node, although I don't know of any authoring tools that would create them that way, or could ingest them that way.

@bghgary assuming this seems OK to you as well i'm fine to make the spec more strict.

Reply:
Seems okay to me.

Reply:
I'd have to review some of the discussion around the "skeleton" (particularly, the extensive information in 1270). But doesn't the statement

> "The node must be the closest common root of the joints hierarchy or a direct or indirect parent node of the closest common root."

boil down to something like "The node must be a common ancestor of all joints"? When it can be a direct or indirect *parent* of the closest common root, then the part of the statement that it must be the *closest common root* does not seem to make sense. 

(Sorry, maybe I overlooked or misunderstood something here...)

Reply:
@donmccurdy @bghgary 
Any opinions on wording here wrt @javagl comment? I'm fine either way.

Reply:
I think both versions read okay to me. @lexaknyazev version calls out more explicitly that it doesn't have the be the _closest_ common root. I'm reading the statement as "The node be [A] or [B]" so they don't conflict with each other.

Reply:
As mentioned in the comment below: I think that both wordings technically mean the same thing, so it may be just a matter of subjective preference.